src/style.css

```diff
@@ -0,0 +1,118 @@
+/* Styles go here */
+:root {
+  --high-blue-color: #060b35;
+  --high-grey-color: #616070;
+  --primary-color: #00acdc;
+  --white-color: #fff;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image img {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 0;
```

Comment:
you're playing a lot with custom margins around each card child element
consider providing paddings for the card itself (32px 16px 16px)
- then consistently apply only margin-bottom for children elements